storage/database/schema.go

```diff
@@ -50,6 +50,22 @@ var (
 
 	snapshotKeyPrefix = []byte("snapshot")
```

Comment:
```suggestion
	// snapshotKeyPrefix tracks Istanbul snapshot.
	snapshotKeyPrefix = []byte("snapshot")
```
The two snapshot is confusing a little.

Reply:
```suggestion
	IstanbulSnapshotKeyPrefix = []byte("snapshot")
```
What do you think of putting `istanbul`?